downstream/modules/platform/ref-removing-instances.adoc

```diff
@@ -0,0 +1,33 @@
+[id="ref-removing-instances"]
+
+= Removing Instances
+
+From the *Add Instances page*, you can add, remove or run health checks on your nodes.
+
+[NOTE]
+====
+You must follow the procedures for installing RHEL packages for any additional nodes you create. 
+If you peer this additional node to an existing hop node, you must also install the Install Bundle on each node.
+====
+
+Use the check boxes next to an instance to select it to remove or run a health check against it. 
```

Comment:
I wonder if this could include some commas:?
Use the check boxes next to an instance to select it, to remove it, or run a health check against it